### ISSUE TITLE
PDE integration: resolving of maven dependency doesn't use configured proxy (+ can't force update refresh)

### ISSUE THREAD

**Issue by SVNKoch** [eclipse-m2e/m2e-core#1754]
In eclipse I have configured a proxy:
![grafik](https://github.com/eclipse-m2e/m2e-core/assets/39488412/b2a04701-1251-4153-9b46-e00979b2e59d)

These settings are however not regarded when fetching a maven dependency in the target definition:
![grafik](https://github.com/eclipse-m2e/m2e-core/assets/39488412/4d01c7c5-090a-476a-ab9b-deb144f5c1d0)

Example uses:
```
<dependency>
    <groupId>com.fasterxml.jackson.core</groupId>
    <artifactId>jackson-core</artifactId>
    <version>2.17.1</version>
</dependency>
```
and results in:
> Could not transfer artifact com.fasterxml.jackson.core:jackson-core:jar:2.17.1 from/to central (https://repo.maven.apache.org/maven2): Connect to repo.maven.apache.org:443 [repo.maven.apache.org/151.101.0.215, repo.maven.apache.org/151.101.64.215, repo.maven.apache.org/151.101.128.215, repo.maven.apache.org/151.101.192.215] failed: Connection refused: no further information

<br><br><br><br>

Also:
when retrying (even without proxy, so that the adresses are indeed reachable), I get:
> com.fasterxml.jackson.core:jackson-core:jar:2.17.1 failed to transfer from https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced. Original error: [...]

(which might be its own issue: Update / Refresh button don't force an update. Couldn't find a way to force it using the UI)

CC @laeubi with PR https://github.com/eclipse-m2e/m2e-core/pull/21 (and thanks for this great integration!)

**Comment by laeubi** [comment]
You most probably need to configure proxies in maven:
https://maven.apache.org/guides/mini/guide-proxies.html

**Comment by SVNKoch** [comment]
thanks, I think this is the issue.

I added the proxy to the .m2/settings.xml file.

However now I get the follwong error instead:
> Could not transfer artifact com.fasterxml.jackson.core:jackson-core:jar:2.3.0 from/to central (https://repo.maven.apache.org/maven2): PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

I made sure / tried:
- maven uses my installed java 17 installation
- in eclipse the chose JRE is the java 17 installation
- in eclipse maven configured to use my locally installed maven
- wen to "https://repo.maven.apache.org/maven2" in my browser, downloaded the certificate (company CA) and imported it into java keystore using `keytool -cacerts -importcert -file "C:\company-cert.crt" -alias "company-cert" -storepass changeit`
- later also tried adding the issuing CA and proxy certs (downloaded them from chain in firefox, then imported as above)
- restarted Eclipse

Maybe you have an idea why this is?
